### DISCUSSION TITLE
I cannot login in the game client. Able to login via website

### DISCUSSION THREAD

**Discussion by simonvaly** [chromiecraft/chromiecraft#6516] [General]
Hello.
I am not able to login today in the game client, even after deleting Cache, Logs and WTF folders and restarting the game. I even changed the password via the website.
Please do help me. Thank you!

**Comment by 55Honey** [comment]
Hello @simonvaly ,
Thanks for reaching out.
There are a couple of possible causes for this behaviour.
1) You're connecting to the wrong server. Make sure both your config.wtf and realmlist.wtf both point to chromieCraft and make sure you're not using a modified client 
2) It appears your account has been banned. Did you manage to login at least once before the error? If not, this is unlikely.

Feel free to open a ticket on our [discord](https://discord.gg/chromiecraft), so you don't need to provide your account details in a public space.

Regards

**Reply by simonvaly** [reply]
I have not been able to login today.
I did checked realmlist.wtf and is corect.
![image](https://github.com/chromiecraft/chromiecraft/assets/57768006/ed08e9c5-5a76-4964-8282-be52406ab932)
Would it be possible I am banned? If so, what could be the reason.

And about the discord
![image](https://github.com/chromiecraft/chromiecraft/assets/57768006/79136b3e-bb93-47f5-a274-c7d3cecad790)
Everything seems wrong today ...

Thanks again.

**Comment by 55Honey** [comment]
Solved in a ticket on discord. :)
